src/analysis/connectivity/overall_scores.sql

```diff
@@ -349,3 +350,32 @@ SELECT  'overall_score',
 -- normalize
 UPDATE  generated.neighborhood_overall_scores
 SET     score_normalized = score_original * :total;
+
+-- population
+INSERT INTO generated.neighborhood_overall_scores (
+    score_id, score_original, human_explanation
+)
+SELECT  'population_total',
+        (
+            SELECT SUM(pop10) FROM neighborhood_census_blocks
+            WHERE   EXISTS (
+                        SELECT  1
+                        FROM    neighborhood_boundary AS b
+                        WHERE   ST_Intersects(b.geom,neighborhood_census_blocks.geom)
+                    )
+        ),
+        'Total population of boundary';
```

Comment:
Can you also update the json file so that they contain these new scores, along with the description text?